cukinia/common_security_tests.d/kernel.conf

```diff
@@ -32,5 +32,37 @@ as "SEAPATH-00174 - Randomize kstack offset in on" cukinia_cmd \
 as "SEAPATH-00175 - Disable slab usercopy fallback" cukinia_cmd \
     grep -q "slab_common.usercopy_fallback=N" /proc/cmdline
 
+#ANSSI-BP-028-R8 l1tf test made in spectre_mitigations.conf
+
+as "ANSSI-BP-028-R8-1 - page_poison is enabled" cukinia_cmd \
+	grep -q "page_poison=on" /proc/cmdline
+
+as "ANSSI-BP-028-R8-2 - pti is enabled" cukinia_cmd \
+	grep -q "pti=on" /proc/cmdline
+
+as "ANSSI-BP-028-R8-3 - slab_nomerge is enabled" cukinia_cmd \
+	grep -q "slab_nomerge" /proc/cmdline
+
+as "ANSSI-BP-028-R8-4 - slub_debug is configured" cukinia_cmd \
+	grep -q "slub_debug=FZP" /proc/cmdline
+
+as "ANSSI-BP-028-R8-5 - spec_store_bypass_disable is enabled" cukinia_cmd \
+	grep -q "spec_store_bypass_disable=seccomp" /proc/cmdline
+
+as "ANSSI-BP-028-R8-6 - spectre_v2 protection is enabled" cukinia_cmd \
+	grep -q "spectre_v2=on" /proc/cmdline
```

Comment:
As Mathieu stated, I think verifying that mitigation is in place in any way through /sys/cpu/vulnerabilities (already existing test) is better:
 - It guarantees we are not vulnerable
 - If the CPU already implements a defense, we won't add additional software defenses which degrade performance.

So for each CPU vulnerability, use the sysfs test if available rather than cmdline.